src/App.jsx

```diff
@@ -14,70 +15,76 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
+  const styleForTable = 'has-background-success-light';
+  const styleForButton = 'button is-info';
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      <h1
+        className="title is-flex is-align-items-center"
+      >
+        {selectedGood === ''
+          ? 'No goods selected'
+          : `${selectedGood} is selected`
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+        }
+        {selectedGood !== ''
+        && (
+        <button
+          data-cy="ClearButton"
+          type="button"
+          className="delete ml-3"
+          onClick={() => setSelectedGood('')}
+        />
+        )
+        }
+      </h1>
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
+      <table className="table">
+        <tbody>
+          {goods.map(good => (
+            <tr
+              data-cy="Good"
+              className={
+                good === selectedGood
+                && `${styleForTable}`
+              }
```

Comment:
there should be key
also `good === selectedGood` repeated several times, try to avoid code repeatability, create some variable which will return `true` or `false` instead of this